src/types/index.ts

```diff
@@ -14,3 +14,18 @@ export interface SimplifiedLayout {
   nodes: (ChartNode | TreeNode)[]
   links: any[]
 }
+
+export interface LinkProps {
+  Component: any // TODO types
```

Comment:
you can do it right now since we expose it : 
`React.Component<LinkProps> | React.FunctionnalComponent<LinkProps>`
or if this does not work : `React.Component | React.FunctionnalComponent`